SPECS/xapi.spec

```diff
@@ -423,6 +424,9 @@ Coverage files from unit tests
 %endif
 
 %changelog
+* Tue Mar 17 2020 Benjamin Reis <benjamin.reis@vates.fr> - 1.160.2-1.1
```

Comment:
The version should be 1.214.1-2.3 (it must match Version and Release). You also need to increase the Release tag.